skbio/io/format/tests/test_genbank.py

```diff
@@ -0,0 +1,429 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+from future.builtins import map, zip
+
+import io
+import numpy as np
+import pandas as pd
+import numpy.testing as nptest
```

Comment:
Can you please use the following form? This is what we use in other test code in skbio (with the exception of one other test file).

```python
import numpy.testing as npt
```